lib/python/ensembl/brc4/runnable/process_seq_region.py

```diff
@@ -382,6 +421,39 @@ def get_codon_table(self, record: SeqRecord) -> int:
                 break
         
         return table
+    
+    def get_organelle(self, record: SeqRecord) -> str:
+        """Given a genbank record, look for an organelle field.
+
+        Args:
+            record: The GenBank record to look into.
+
+        Returns:
+            the organelle location (empty string if not found).
+        """
+
+        location = 0
+        molecule_location = self.param("molecule_location")
+
+        for feat in record.features:
+            if "organelle" in feat.qualifiers:
+                organelle = str(feat.qualifiers["organelle"][0])
+                if not organelle:
+                    break
+
+                # Remove plastid prefix
+                with_prefix = re.match(r'^(plastid|mitochondrion):(.+)$', organelle)
+                if with_prefix:
+                    organelle = with_prefix[2]
+
+                # Get controlled name
+                try:
+                    location = molecule_location[organelle]
+                except KeyError:
+                    raise KeyError(f"Unrecognized sequence location: {organelle}")
+                break
+        
+        return location
```

Comment:
Unless I'm missing something, this does not match exactly with the docstring. Either you need to add a `Raises:` section in the docstring, or you may choose to do:
```suggestion
                return molecule_location.get(organelle, "")
```